x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/target_field_help_text.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiText } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  getMlModelTypesForModelConfig,
+  SUPPORTED_PYTORCH_TASKS,
+} from '../../../../../../../common/ml_inference_pipeline';
+import { TrainedModel } from '../../../../api/ml_models/ml_trained_models_logic';
+import { getMLType } from '../../../shared/ml_inference/utils';
+
+export interface TargetFieldHelpTextProps {
+  model?: TrainedModel;
+  pipelineName: string;
+  targetField: string;
+}
+
+export const TargetFieldHelpText: React.FC<TargetFieldHelpTextProps> = ({
+  pipelineName,
+  targetField,
+  model,
+}) => {
+  const baseText =
+    targetField.length > 0
```

Comment:
nitpick(non-blocking): I like falsy checks more than explicit string length checks, because if targetField turns out to be accidentally undefined/null, falsy checks behave as expected while string length checks throw with unpredictable results.